style-guides/ruby/rails/controllers/README.md

```diff
@@ -0,0 +1,19 @@
+# Ruby on Rails Controllers
+
+## Responders
+
+* Use responders [plataformatec/responders](https://github.com/plataformatec/responders) to dry up your Rails app (3.2+). Works with :html and :json
```

Comment:
Victor this is a RubyOnRails best-practice, so keep it in: best-practices/rubyonrails/controllers/README.md